2023-01-21 12:21:26.021 | ERROR    | __main__:_make_request:32 - Erro de conexão ao fazer %s request para %s: %s
2023-01-21 12:22:43.123 | ERROR    | __main__:_make_request:32 - Erro de conexão ao fazer GET request para /user: No connection adapters were found for '127.0.0.1:8000/api/v1//user'
2023-01-21 12:22:49.932 | ERROR    | __main__:_make_request:32 - Erro de conexão ao fazer GET request para user: No connection adapters were found for '127.0.0.1:8000/api/v1/user'
2023-01-21 12:23:35.746 | ERROR    | __main__:_make_request:41 - Erro ao fazer GET pedido para /usuario: {'detail': 'As credenciais de autenticação não foram fornecidas.'} (Erro de codigo 403)
